modules/client/src/multisig/internal/utils.ts

```diff
@@ -98,3 +102,54 @@ export function toMultisigProposalListItem(
     status: computeProposalStatus(proposal),
   };
 }
+
+export function computeProposalStatus(
+  proposal: SubgraphMultisigProposal | SubgraphMultisigProposalListItem,
+): ProposalStatus {
+  const now = new Date();
+  const startDate = new Date(
+    parseInt(proposal.startDate) * 1000,
+  );
+  const endDate = new Date(parseInt(proposal.endDate) * 1000);
+  if (proposal.executed) {
+    return ProposalStatus.EXECUTED;
+  }
+  if (now < startDate) {
+    return ProposalStatus.PENDING;
+  }
+  if (proposal.potentiallyExecutable && now < endDate) {
```

Comment:
Are we expressly not taking into account the `minApproval` = 1 case? All non Pending proposals will be successful by default for a plugin with `minApproval` = 1

Reply:
Its not the intended behaivour? I mean if you create a proposal witn minApprovals = 1, any membber of the multisig can just approve an execute, we can add the check to see if the proposal minApproval is 1 and the length of approvals is 0 to return it as Active

Reply:
I will leave this to your discretion